advanced-achievements-plugin/src/main/resources/lang-FR.yml

```diff
@@ -1,14 +1,17 @@
 #=====================================================================#
 # +-----------------------------------------------------------------+ #
 # | Advanced Achievements French translation by DarkPyves/Taehyuun  | #
-# |                 ~ Valid as of version 5.2.7 ~                   | #
+# |                 ~ Valid as of version ${project.version} ~                   | #
```

Comment:
Could you please indicate 5.15.0 here? When new language elements are added but not translated, this will still indicate the right version for which this translation is valid.